src/aleph/sdk/conf.py

```diff
@@ -92,6 +92,12 @@ class Settings(BaseSettings):
     )
     SCHEDULER_URL: ClassVar[str] = "https://scheduler.api.aleph.cloud/"
 
+    VOUCHER_METDATA_TEMPLATE_URL: str = (
+        "https://claim.twentysix.cloud/sbt/metadata/{}.json"
+    )
+    VOUCHER_SOL_REGISTRY: str = "https://api.claim.twentysix.cloud/v1/registry/sol"
+    VOUCHER_SENDER: str = "0xB34f25f2c935bCA437C061547eA12851d719dEFb"
```

Comment:
I think that instead `VOUCHER_SENDER` should be something like `VOUCHER_ORIGIN_ADDRESS`
```suggestion
    VOUCHER_ORIGIN_ADDRESS: str = "0xB34f25f2c935bCA437C061547eA12851d719dEFb"
```